.github/workflows/php-test.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - trunk
       - 'release/**'
+      - 'feature/**'
```

Comment:
There is probably no need in running php tests on every push to a feature branch because all tests will be executed when we create a PR for that feature against the `trunk` branch.
```suggestion
```